content/about/index.md

```diff
@@ -10,4 +10,32 @@ Packit provides tooling and automation to integrate upstream open source project
 
 https://github.com/packit-service/packit
 
+
+### The key principles of packit
+
+* Reuse existing tools and services: don't reinvent the wheel.
+
+* You can consume packit in two forms:
+  * Packit tool can run on your laptop and you run commands you want.
+  * Packit service reacts to events and performs actions which you
+    defined in your `packit.yaml`.
+
+* We don't break backwards compatibility just like that.
+
+* Packit has a deprecation policy:
+  * We can mark a behaviour as deprecated.
+  * Deprecated content can be removed or changed after at least 3 minor
+    releases.
+  * Deprecated content is advertised in our
+    [changelog](https://github.com/packit-service/packit/blob/master/CHANGELOG.md),
+    printed in terminal or tracked in your issue tracker.
+
+* We care about artifacts which Fedora supports: at the moment it's RPMs,
+  modules and container images.
+  * Our initial focus is solely on RPMs.
```

Comment:
Digging. The initial focus is RPMs, but generally, it works also in any upstream repo, including containers.

Reply:
Yes, but the focus is RPMs: the fact that one or two commands work well with other formats is bonus points for us, but we can't build container images for example.